loop/chain.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "loop",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.loop.fans/",
+  "pretty_name": "Loop",
+  "chain_id": "loop-1",
+  "bech32_prefix": "loop",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
https://github.com/satoshilabs/slips/pull/1709